cve_bin_tool/cve_scanner.py

```diff
@@ -179,19 +180,25 @@ def get_cves(self, vendor: str, product: str, version: str, remarks: Remarks):
             else:
                 self.products_without_cve += 1
             self.all_cves.add(cve_data)
-
-    @staticmethod
-    def openssl_convert(version: str) -> str:
+        else:
```

Comment:
```suggestion
        else:
            # cve_list was empty, so no CVEs were found
```

This else is pretty far away from its if, we might as well be explicit about what this case is falling through from.  Not sure that my suggestion text is the best, but something like that?